src/codec/field_codec.h

```diff
@@ -38,216 +37,10 @@ using ::openmldb::type::DataType;
 /**
  *  encode part
```

Comment:
delete the annotation as well

Reply:
done!

Reply:
please delete the following annotation
https://github.com/4paradigm/OpenMLDB/blob/cfe800638f5443465a078da07d86dbd962682e40/src/codec/field_codec.h#L37-L43

Reply:
> please delete the following annotation
> 
> https://github.com/4paradigm/OpenMLDB/blob/cfe800638f5443465a078da07d86dbd962682e40/src/codec/field_codec.h#L37-L43

done